### DISCUSSION TITLE
I saw somewhere a material layout diagram, but where?

### DISCUSSION THREAD

**Discussion by nielsnl68** [squidfunk/mkdocs-material#6496] [Q&A]
In the last couple of days, i have been looking around of the site and this repo and at some point i saw a very interesting diagram on how this theme was setup with all the blocks that could be used and modified,
Sadly, after i saw it. i closed that page and i cant remember where i saw it.
Does anyone know where i have seen it, thanks?

**Comment by SaltyAimbOtter** [comment]
Could it be this view on the [customization page](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme)?

**Comment by nielsnl68** [comment]
sadly not.
It did look like this, but bigger and more detailed:
```
+-----------------------------+
|           Header            |
+-----+-----------------+-----+
|  S  |       Hero      |     |
|  i  +-----------------+  T  |
|  t  |                 |  O  |  
|  e  |     Content     |  C  |  
|  b  |                 |     |  
|  a  +-----------------+-----+  
|  r  |       Footer          | 
+-----+-----------------------+
|          Main footer        |
+-----------------------------+
   
```

**Reply by squidfunk** [reply]
Having written most of the docs, I'm quite confident to say that we don't have such a diagram.

**Comment by nielsnl68** [comment]
Okay, thanks for your answer. Any change you have seen it anywhere else in the mkdocs world?
